torch/csrc/jit/codegen/cuda/transform_replay.cpp

```diff
@@ -644,24 +644,173 @@ std::pair<TensorDomain*, unsigned int> TransformReplay::replayCasP(
   return replayCasP(consumer, producer, compute_at_axis, root_map);
 }
 
+namespace {
+
+int getMatchedLeafPosWithoutReplay(
+    const TensorView* producer,
+    const TensorView* consumer,
+    int consumer_or_producer_pos,
+    bool consumer_pos = true) {
```

Comment:
We often use names like `consumer_pos` to indicate a position in a consumer domain, so this could be confusing. Maybe something like `is_producer_as_consumer`?

Reply:
This part of the code has been replace by https://github.com/csarofeen/pytorch/pull/1791, no more such issue 😉